src/tester/test.ts

```diff
@@ -39,6 +39,10 @@ const tester = async (): Promise<void> => {
 
         await adapter.run();
 
+        if (entities.length == 0) {
```

Comment:
I don't think you will ever get this error, because it's waiting until at least 4 entities are found before starting tests.

Reply:
I had this issue before migrating to typescript, where I used a block close to the current height and no events were triggered. It had passed even though it had nothing to test, maybe we should leave this just to be cautious?